app/helpers/claims_helper.rb

```diff
@@ -51,8 +51,7 @@ def date_of_birth_string(claim)
     claim.date_of_birth && l(claim.date_of_birth)
   end
 
-  def additional_payments_open?
-    policy = PolicyConfiguration.policy_for_routing_name("additional-payments")
-    PolicyConfiguration.for(policy).open_for_submissions?
+  def additional_payments_open?(current_policy)
+    PolicyConfiguration.for(current_policy).open_for_submissions?
```

Comment:
This method is specifically for the `additional-payments` policy so this might lead to unexpected behaviour.

Reply:
Do the other policies operate within open windows?

Reply:
For context we get this error in`./spec/features/teacher_detail_page_student_loans_journey_spec.rb` when we have the original implementation:

```
     ActionView::Template::Error:
       undefined method `open_for_submissions?' for nil:NilClass
     # ./app/helpers/claims_helper.rb:57:in `additional_payments_open?'
```

We could tidy up in this PR or address this whole area properly in the PR for the tslr landing page [CAPT-1353](https://dfedigital.atlassian.net/browse/CAPT-1353) "Dev - TSLR ID - New landing page".

What do you think?

Reply:
@your Thanks for the comment. I'm not keen on adding unrelated bits to the specs, but I'm prepared to disagree and commit if this is the preference?